apps/passport-client/components/screens/SecondPartyTicketVerifyScreen.tsx

```diff
@@ -166,20 +365,12 @@ function VerifiedAndKnownTicket({
  *
  * Returns a {@link VerifyResult}
  */
-async function deserializeAndVerify(
-  pcdStr: string,
-  pcds: PCDCollection
-): Promise<VerifyResult> {
-  // decodedPCD is a JSON.stringify'd {@link SerializedPCD}
-  const decodedPCD = decodeQRPayload(pcdStr);
-
+async function verify(pcd: PCD, serializedPCD: string): Promise<VerifyResult> {
   const result = await requestVerifyTicket(appConfig.zupassServer, {
-    pcd: decodedPCD
+    pcd: JSON.stringify(serializedPCD)
```

Comment:
Ah, the type here is incorrect (it passed type-checking because the `JSON.parse()` on line 346 returns an `any` type. With the fixed type of `SerializedPCD`, the `stringify` makes sense.